wordpress-mobile/gutenberg-mobile
# Add/experimental block grid block

Block Experiments PR: https://github.com/Automattic/block-experiments/pull/136 (Updates to the layout grid blocks edit) 

Here we are adding the `block-experiments` [repo](https://github.com/Automattic/block-experiments/) that includes the layout-grid block. In order to add support for that block in the gutenberg mobile editor.

This PR just add the basic read only block and doesn't include support for all the different block attributes that the layout block current supports. 

This PR add the following:
1. The block-experiments repo as a submodule. 
2. The layout-grid block as a block so that it can be added to the block editor.

**How this PR has been tested.** 
It has been tested on the demo app only.

To test:
Add the new repo:
```
git submodule init
git submodule update
```

Start the demo app
```
npm run start:reset
npm run core ios
```

Now you should be able to add the layout grid block to the demo app. 
See
<img src="https://user-images.githubusercontent.com/115071/112394724-6e768980-8cba-11eb-9a00-6ec2659237c0.png?s=200" width="300" />


After it is inserted it looks like this. 
<img src="https://user-images.githubusercontent.com/115071/112394670-5d2d7d00-8cba-11eb-8e58-b8b7eb7b06bf.png?s=200" width="300" />


PR submission checklist:
